### PR TITLE
[Az.RecoveryServices.backup] Added support for MUA new operations

### PR DESCRIPTION
**Pull request by hiaga**
## Description

Added support for MUA new operations - Disable vault immutability, modify schedule policy RPO, Restore, Stop protection and retain forever. 

## Mandatory Checklist

- Please choose the target release of Azure PowerShell
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by Nickcandy** [comment]
@hiaga Hi, please check the status of ci [pipeline](https://dev.azure.com/azclitools/public/_build/results?buildId=150537&view=logs&j=b9afb22f-0837-5d02-bc24-6c30aee08bcf&t=990f6f37-4a21-5927-092a-220f38a460e9). There are some errors about the parameter and the tests seem failed. Please fix it. Just a friendly reminder: our code freeze deadline is at 6 PM PST on April 22nd. If your code is intended for this release, please ensure it is completed as soon as possible.

**Comment by hiaga** [comment]
working on CI failure, will resolve shortly

File: src/RecoveryServices/RecoveryServices.Management.Sdk/README.md [hunk 1]
```diff
@@ -23,12 +23,12 @@ payload-flattening-threshold: 2
...
 ###
 ``` yaml
-commit: 8d1d3d29cdbe03f95a9b4c0ec6bb2221f1c44bde
+commit: e6a20fec72ed3bcb4b43c559ee20b56ca2786ec0
 input-file:
-  - https://github.com/Azure/azure-rest-api-specs/blob/$(commit)/specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2024-02-01/registeredidentities.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/$(commit)/specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2024-02-01/replicationusages.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/$(commit)/specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2024-02-01/vaults.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/$(commit)/specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2024-02-01/vaultusages.json
+  - https://github.com/hiaga/azure-rest-api-specs/blob/$(commit)/specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2024-04-01/registeredidentities.json
```
Comment (Nickcandy) [inline src/RecoveryServices/RecoveryServices.Management.Sdk/README.md:28]: Please change it to Azure repo and regenerate

**Review by Nickcandy — APPROVED** [review]

**Pull request merged**